src/conn/conn_api.c

```diff
@@ -1450,9 +1450,20 @@ __conn_rollback_to_stable(WT_CONNECTION *wt_conn, const char *config)
     WT_CONNECTION_IMPL *conn;
     WT_DECL_RET;
     WT_SESSION_IMPL *session;
+    char config_buf[16];
 
     conn = (WT_CONNECTION_IMPL *)wt_conn;
 
+    /*
+     * If config is NULL or empty, and threads_num is configured at the connection level, use the
```

Comment:
How about `In the absence of an API configuration, utilize the RTS worker thread settings defined at the connection level.`